rj_gameplay/reg_pass_costing/cost_heuristic.py

```diff
@@ -0,0 +1,157 @@
+import stp.rc as rc
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+import numpy as np
+
+MAX_PASS_DIST = 5
+MAX_GOAL_DIST = 7.5
+MAX_AVG_OPP_DIST = 6
+MAX_NEAREST_OPP_DIST = 2.5
+MAX_OPP_BETWEEN_ROBOT_GOAL = 5
+MAX_OPP_BETWEEN_ROBOT_BALL = 1
+MIN_GOAL_DIST_BONUS = 1
+MAX_GOAL_DIST_BONUS = 1.75
+
+GOAL_DIST_COST = 150
+AVG_OPP_DIST_COST = 75
+NEAREST_OPP_DIST_COST = 300
+OPP_BETWEEN_ROBOT_GOAL_COST = 325
+OPP_BETWEEN_ROBOT_BALL_COST = 650
+
+MAX_COST = 1000
+
+def cost_heuristic(world_state: rc.WorldState, robot: rc.Robot) -> float:
+
+    field = world_state.field
+
+    our_robots = world_state.our_robots
+    their_robots = world_state.their_robots
+
+    robot_with_ball = our_robots[0]
+    for bot in our_robots:
+        if bot.has_ball_sense:
+            robot_with_ball = bot
+            if robot_with_ball == robot:
+                return -1
```

Comment:
This should be a really big number right? You wouldn't pass to a robot which already has the ball.